src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -1514,13 +1514,20 @@ public FeedbackSessionDetailsBundle getFeedbackSessionDetails(
             break;
 
         case PRIVATE:
-            if (fqLogic.getFeedbackQuestionsForSession(
-                    fsa.feedbackSessionName, fsa.courseId).isEmpty()) {
+            List<FeedbackQuestionAttributes> instuctorQuestions =
+                    fqLogic.getFeedbackQuestionsForInstructor(
+                            fsa.feedbackSessionName,
+                            fsa.courseId,
+                            fsa.creatorEmail);
+            List<FeedbackQuestionAttributes> validQuestions = fqLogic
+                    .getQuestionsWithRecipients(instuctorQuestions,
+                            fsa.creatorEmail);
+            if (validQuestions.isEmpty()) {
                 break;
             }
             details.stats.expectedTotal = 1;
-            if (this.isFeedbackSessionFullyCompletedByInstructor(
```

Comment:
I am not using this function because using it in its current form will lead to redundant db operations.